osu.Game/Overlays/Chat/ChatTextBar.cs

```diff
@@ -141,16 +141,16 @@ protected override void LoadComplete()
 
                 switch (newChannel?.Type)
                 {
-                    case ChannelType.Public:
-                        chattingText.Text = ChatStrings.TalkingIn(newChannel.Name);
+                    case null:
+                        chattingText.Text = string.Empty;
                         break;
 
                     case ChannelType.PM:
                         chattingText.Text = ChatStrings.TalkingWith(newChannel.Name);
                         break;
 
                     default:
-                        chattingText.Text = string.Empty;
+                        chattingText.Text = ChatStrings.TalkingIn(newChannel.Name);
```

Comment:
I guess this change is to accommodate announcement channels but I'm not sure it even makes sense? Like are announcement channels full-fledged channels, or are they "read-only" and users that receive announcements cannot respond to them?

Reply:
[From an earlier conversation in discord](https://discord.com/channels/188630481301012481/188630652340404224/979679073246789672) it was determined that only some users have write access to announce channels, but there's currently no way to check for that yet so I determined it best to allow the chat text box to appear correct for the channel and allow for normal error handling if a user tries to send a message to a channel they don't have permissions for.